assets/scss/base/_buttons.scss

```diff
@@ -49,6 +49,11 @@ button, .button {
   margin-top: 10px;
 }
 
+.button--secondary-small {
```

Comment:
extending button-secondary won't work as the bkg-color will be green?
ok with the following?:

.button--secondary-small {
  @extend .button--small;
  @include button($grey-8);
}